setup.py

```diff
@@ -55,14 +55,22 @@
         "pyarrow",
         "tldextract",
         "diskcache",
+        "requests_viewer",  # utils
         "tzlocal",
-        # "lxml",  # web_history
-        # "diskcache",  # web_history
-        # "auto_extract",  # web_history
-        # "ujson",  # web_history
-        # "google-api-python-client",  # offers
-        # "python-dotenv",  # offers
-        # "natura",  # offers
+        "lxml",  # web_history
```

Comment:
I'd recommend against comments like this in the requirements.  They're nice right at this moment, but they're just waiting to become stale if another source uses a particular requirement.  If we want to to keep track of modular requirements, we'd be much better off recording them another way.  (E.g., we could have a json file with requirements listed for each data source, or a requirements.txt file in every data source's folder.)

Reply:
Yea - that is the proper way in the future. You are totally right they will be stale so it is kinda useless!